internal/venice-common/src/test/java/com/linkedin/venice/utils/UtilsTest.java

```diff
@@ -1,9 +1,6 @@
 package com.linkedin.venice.utils;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
-import static org.testng.Assert.expectThrows;
-import static org.testng.Assert.fail;
+import static org.testng.Assert.*;
```

Comment:
Can you please remove wildcard imports? If you're using intellij, try this: https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports

The rest LGTM